test/framework/filetools.py

```diff
@@ -793,6 +793,12 @@ def test_apply_patch(self):
         pattern = "I'm a toy, and very proud of it"
         self.assertTrue(pattern in patched)
 
+        toy_patch_gz = os.path.join(testdir, 'sandbox', 'sources', 'toy', 'toy-0.0_gzip.patch.gz')
+        self.assertTrue(ft.apply_patch(toy_patch_gz, path))
```

Comment:
ah, I missed the `very very` bit, ok, fine by me, but let's add a comment then to make it clear that the zipped patch is an incremental one